ruby/rcloadenv.gemspec

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+;
+
+lib = File.expand_path "../lib", __FILE__
+$LOAD_PATH.unshift lib unless $LOAD_PATH.include? lib
+require 'rcloadenv/version'
+
+::Gem::Specification.new do |spec|
+  spec.name = "rcloadenv"
+  spec.version = ::RCLoadEnv::VERSION
+  spec.authors = ["Daniel Azuma"]
+  spec.email = ["dazuma@gmail.com"]
```

Comment:
Why not @google.com? Asking based on [python](https://github.com/GoogleCloudPlatform/rcloadenv/blob/master/python/setup.py#L34)